src/system/MailerModule/Resources/config/services.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="zikula_mailer_module.container.link_container.class">Zikula\MailerModule\Container\LinkContainer</parameter>
+        <parameter key="zikula_mailer_module.form.config_type.class">Zikula\MailerModule\Form\Type\ConfigType</parameter>
+        <parameter key="zikula_mailer_module.form.test_type.class">Zikula\MailerModule\Form\Type\TestType</parameter>
+        <parameter key="twig.extension.zikula_mailer_module.class">Zikula\MailerModule\Twig\TwigExtension</parameter>
+        <parameter key="zikula_mailer_module.api.mailer.class">Zikula\MailerModule\Api\MailerApi</parameter>
+    </parameters>
+
+    <services>
+        <service id="zikula_mailer_module.container.link_container" class="%zikula_mailer_module.container.link_container.class%">
+            <argument type="service" id="translator.default" />
+            <argument type="service" id="jms_i18n_routing.router" />
+            <argument type="service" id="zikula_permissions_module.api.permission" />
+            <tag name="zikula.link_container"/>
+        </service>
+
+        <service id="zikula_mailer_module.form.config_type" class="%zikula_mailer_module.form.config_type.class%">
+            <argument type="service" id="translator.default" />
+            <argument type="service" id="zikula.dynamic_config_dumper" />
+            <argument type="service" id="zikula_extensions_module.api.variable" />
+            <tag name="form.type" />
+        </service>
+
+        <service id="zikula_mailer_module.form.test_type" class="%zikula_mailer_module.form.test_type.class%">
+            <argument type="service" id="translator.default" />
+            <argument type="service" id="zikula.dynamic_config_dumper" />
+            <argument type="service" id="zikula_extensions_module.api.variable" />
+            <tag name="form.type" />
+        </service>
```

Comment:
I dislike making all formTypes services because they are not used very often in the application (how often do you adjust the mailer settings, once?) but making them a service always loads them into memory and _implies_ that it is something that is used often. At the very least, it should [lazy loaded](http://symfony.com/doc/current/components/dependency_injection/lazy_services.html) (e.g. `lazy="true"`), but I suggest reconsidering how to utilize the form without making it a service.

I know the argument is because of the dependent services. You will see that the way I have done them in other modules is to add the translator in the `$options` array. This is also done with other 'dependencies'.

Anyways - give it some thought.